api/app/signals/apps/sigmax/templates/sigmax/pdf_template.html

```diff
@@ -89,6 +89,18 @@ <h2>Melder</h2>
     </tr>
 </table>
 
+{% if signal.extra_properties %}
+    <h2>Extra eigenschappen</h2>
+    <table class="table">
+        {% for key, value in signal.extra_properties.items %}
```

Comment:
Huidige versie is niet ge-namespaced. Het voorstel besproken met FE is om alle extra uitvraag onder de sleutel: "extra_uitvraag" te plaatsen (zodat we niet alles wat toevallig in extra_properties herhalen in de PDF.

Reply:
Dit wordt een extra nieuwe ticket, en later opgepakt.